src/compute/src/row_spine.rs

```diff
@@ -99,32 +99,30 @@ mod spines {
 
 /// A `Row`-specialized container using dictionary compression.
 mod container {
-    use differential_dataflow::IntoOwned;
+
     use std::cmp::Ordering;
 
     use differential_dataflow::trace::implementations::BatchContainer;
-    use mz_ore::region::Region;
-    use mz_repr::{read_datum, Datum, Row, RowPacker};
+    use differential_dataflow::IntoOwned;
     use timely::container::PushInto;
 
```

Comment:
```suggestion
```

Nit: `use` formatting.

Reply:
Will clean up in the next PR, if that's ok. Never sure where we want line break/don't (e.g. should line 104 be deleted too?).

Reply:
There is a section about this in our style guidelines that may help against the uncertainty: https://github.com/MaterializeInc/materialize/blob/main/doc/developer/style.md#imports